packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -10,30 +12,69 @@ import NicknameForm from '../nickname/nickname-form.jsx';
 import { buy_sell } from '../../constants/buy-sell';
 import BuySellForm from '../buy-sell/buy-sell-form.jsx';
 import FormError from '../form/error.jsx';
-import { localize } from '../i18next';
+import { localize, Localize } from '../i18next';
 import './advertiser-page.scss';
 
 const RowComponent = React.memo(({ data: advert, showAdPopup, style }) => {
     const { advertiser_page_store, general_store } = useStores();
     const { currency } = general_store.client;
+
     const { local_currency, max_order_amount_limit_display, min_order_amount_limit_display, price_display } = advert;
 
     const is_buy_advert = advertiser_page_store.counterparty_type === buy_sell.BUY;
     const is_my_advert = advertiser_page_store.advertiser_details_id === general_store.advertiser_id;
 
     return (
         <div style={style}>
-            <Table.Row className='advertiser-page__adverts-table_row'>
-                <Table.Cell>
-                    {`${min_order_amount_limit_display}-${max_order_amount_limit_display} ${currency}`}
-                </Table.Cell>
-                <Table.Cell className='advertiser-page__adverts-price'>
-                    {price_display} {local_currency}
-                </Table.Cell>
+            <Table.Row
+                className={classNames('advertiser-page__adverts-table_row', {
+                    'advertiser-page__adverts-table_row--mobile': isMobile(),
```

Comment:
Instead of adding `--mobile` modifiers everywhere you could use mixins from `devices.scss`.

Reply:
This is done, Aaron, thank you! 😄